README.md

```diff
@@ -122,11 +122,37 @@ LOG_LEVEL=trace npm run test
 
 Alternatively, one may set `LOG_LEVEL` in the `.env` file.
 
-### Authentication
-
-The service validates a Bearer JSON Web Token (JWT) found in the `Authorization` HTTP header.
-
-See the `.env.example` for relevant variables.
+### Authentication and Authorization
+
+A valid Bearer JSON Web Token (Bearer JWT) is required in requests to the PDC service. The PDC officially reccomends using [KeyCloak](https://www.keycloak.org/) as the authentication provider, but supports any valid Bearer JWT provider. Please refer to [the relevant documentation for setup and configuration](https://www.keycloak.org/guides#getting-started)
+
+The relevant authorization enviroment variables as they appear in the `.env.example` file are:
+
+- `AUTH_SERVER_ISSUER`: the URL for the issuer (this is the "realm" URL in Keycloak terminology).
```

Comment:
What do you think about moving this explanation to the `.env.example` file itself as a comment before the relevant env variable?

Also, I generally suggest avoiding the documentation of specific env variables in readme documentation since it means you're documenting in two places (or, NOT documenting in the most important place, which is in the `.env.example`). 

If you agree, then you would remove the list of env vars from here and instead make sure they're in a section in the template, and you could just say something like "see `.env.example` for documentation on the necessary environment variables for Keycloak"